test/arrToObj.js

```diff
@@ -0,0 +1,9 @@
+function arrToObj(arr) {
+  const obj = {}
+  for (each of arr) {
```

Comment:
the linter doesn't catch this exactly but we need to avoid newer syntax in order to support more node + browser versions. could we use `arr.forEach`?

Reply:
check here if you're ever curious about syntax, i want this library to support node 6.4

https://node.green/